examples/scale-deployment.js

```diff
@@ -9,16 +10,31 @@ const targetNamespaceName = 'default';
 const targetDeploymentName = 'docker-test-deployment';
 const numberOfTargetReplicas = 3;
 
-async function scale(namespace, name, replicas) {
-  // find the particular deployment
-  const res = await k8sApi.readNamespacedDeployment(name, namespace);
-  let deployment = res.body;
+async function scale(deploymentNamespace, deploymentName, replicas) {
```

Comment:
Can we stick to`namespace` and `name` here instead of `deploymentNamespace` and `deploymentName`?

Reply:
The reason I did this was because as an example, I think it is easier to read the named parameters as `{ name: variableName, namespace: anotherName }` rather than using object shorthand property names like `{ name, namespace }`. In general I prefer what you are saying, but as an example I find this more easier to digest.